piptools/repositories/local.py

```diff
@@ -61,6 +62,10 @@ def finder(self) -> PackageFinder:
     def session(self) -> Session:
         return self.repository.session
 
+    @property
+    def command(self) -> InstallCommand:
```

Comment:
"""Return an install command instance."""

Reply:
Addressed in 81419f9f1090bda0761020e7e9832ba279a5fd4d